src/components/ui/common/Actions.js

```diff
@@ -27,6 +27,7 @@ exports.selectDropdownOption = function( opts ) {
         });
     }).then(function(filtered) {
         filtered[0].click();
+        openBtn.click();
```

Comment:
please add comment what this click does. 
also lets refactor these lines to 
1) use the test filter
2) instead of then do `first().click()` which is shorter and nicer.